pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[project]
+name = "mimir-worker-k8s"
+version = "0.1"  # this is in fact irrelevant
+requires-python = ">=3.12"
```

Comment:
Not sure what is the implication of this field.
We must be consistent with what we provide in the `bases:` section.

Reply:
Without the `require-python` key, we receive this warning:

> warning: No `requires-python` value found in the workspace. Defaulting to `>=3.12`.

Since the `bases` only defines the OS release, how would we compare Python versions for consistency?

Reply:
Will pin to the Python version we use everywhere else https://packages.ubuntu.com/focal/python3